@kiva/kv-components/vue/KvLightbox.vue

```diff
@@ -44,17 +45,19 @@
 							tw-bg-primary
 							tw-flex tw-flex-col
 							tw-mx-auto md:tw-my-auto
+
 						"
 						:class="{
+							'tw-w-full md:tw-w-auto' : variant === 'lightbox',
```

Comment:
This makes it so that when a lightbox has content that is very narrow it is still full width on mobile